x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.tsx

```diff
@@ -377,12 +386,21 @@ export const StepDefineForm: FC<StepDefineFormProps> = React.memo((props) => {
           </EuiFlexGroup>
         ) : null}
         {stepDefineForm.transformFunction === TRANSFORM_FUNCTION.LATEST ? (
-          <LatestFunctionForm latestFunctionService={stepDefineForm.latestFunctionConfig} />
+          <LatestFunctionForm
+            latestFunctionService={stepDefineForm.latestFunctionConfig}
+            copyToClipboard={copyToClipboardPivot}
+            copyToClipboardDescription={copyToClipboardPivotDescription}
```

Comment:
Did you consider passing the "Copy to clipboard" component as a child instead of adding extra props?

Reply:
It's the same prop structure how we pass it on to `DataGrid` so didn't want to change that for this PR.